build-system/common/esbuild-babel.js

```diff
@@ -168,7 +171,9 @@ function getEsbuildBabelPlugin(
 
       const babelOptions =
         babel.loadOptions({caller: {name: callerName}}) || {};
-      const optionsHash = md5({babelOptions, argv});
+      const optionsHash = md5(
+        JSON.stringify({babelOptions, argv: process.argv.slice(2)})
```

Comment:
```suggestion
        JSON.stringify({babelOptions, argv: process.argv})
```

Reply:
That only creates a portability problem, because now the CWD will also be hashed. If I move the `amphtml` directory, I have to recompiled. 🙁